physics/SFC_Models/Land/Noahmp/module_sf_noahmplsm.F90

```diff
@@ -5727,6 +5755,17 @@ subroutine gfs_stability                                              &
 
           endif          ! end of if (dtv >= 0 ) then loop
 !
+! form drag coefficient
+!
+          if ( varf.gt.varf_min .and. psl_gwd_z0m_factor.gt.0.0 ) then
+            zf    = min( min(varf,varf_max)*psl_gwd_z0m_factor,z1 )
```

Comment:
add a comment for what `zf` represents

Reply:
eventually we plan to implement this into the refactored noahmp repository, that will require more descriptive variable names so we need to know what these factors represent

Reply:
since it requires no additional variables and varf is not used below, for clarity I would suggest
```
varf = min(varf, varf_max)
zf = varf * gwd_z0m_factor
zf = min(zf, z1)
```

Reply:
resolved